src/System.Runtime/tests/System/UIntPtrTests.cs

```diff
@@ -138,6 +138,7 @@ public static unsafe void TestImplicitCast()
             Assert.Throws<OverflowException>(() => (uint)ptr);
         }
 
+        [ActiveIssue(20154, TargetFrameworkMonikers.NetFramework)]
```

Comment:
disable for desktop -- this is by design, I see the bug was fixed in UIntPtr.GetHashCode